playbooks/common/openshift-cluster/upgrades/v3_8/upgrade_control_plane.yml

```diff
@@ -14,7 +14,7 @@
 - import_playbook: ../init.yml
   vars:
     l_upgrade_no_switch_firewall_hosts: "oo_masters_to_config:oo_etcd_to_config:oo_lb_to_config"
-    l_upgrade_non_node_hosts: "oo_masters_to_config:oo_etcd_to_config:oo_lb_to_config"
```

Comment:
Not sure if this removal is a good idea - its still being used in `playbook/init/facts.yml` and remains in `playbooks/common/openshift-cluster/upgrades/v3_9/upgrade_control_plane.yml`

Reply:
yeah, that was unintentional, thanks for catching it.